store/tikv/coprocessor.go

```diff
@@ -408,6 +409,14 @@ func (it *copIterator) sendToTaskCh(ctx goctx.Context, t *copTask, taskCh chan<-
 	return
 }
 
+func (it *copIterator) sendToRespCh(goCtx goctx.Context, resp copResponse, respCh chan copResponse) {
+	select {
+	case respCh <- resp:
+	case <-it.finished:
```

Comment:
we should add a mechanism to make the current goroutine exit when `it.finished` is closed

Reply:
Nice catch! I'll fix here.